src/entities/Snapshot/queries.ts

```diff
@@ -1,32 +1,22 @@
-export const DELEGATIONS_ON_PROPOSAL_QUERY = `
-query ($space: String!, $address: String!, $blockNumber: Int) {
-  delegatedTo: delegations(block:{number: $blockNumber},where: { space_in: ["", $space], delegator: $address }, orderBy: timestamp, orderDirection: desc) {
-    delegator
-    delegate
-    space
-    timestamp
-  },
-  delegatedFrom: delegations(block:{number: $blockNumber},where: { space_in: ["", $space], delegate: $address }, orderBy: timestamp, orderDirection: desc) {
-    delegator
-    delegate
-    space
-    timestamp
+type BlockNumber = string | number | undefined
+const getBlockNumberArgument = (blockNumber: BlockNumber) => (blockNumber ? '$blockNumber: Int' : '')
```

Comment:
I think it's not necessary to return this as an empty string, because the `Int` without a bang `!` means it can be undefined.

Might just always work with `'$blockNumber: Int'`

Reply:
True! 🙌